api/stripe.go

```diff
@@ -226,19 +225,19 @@ func (api *API) processStripeSub(ctx context.Context, s *stripe.Subscription) er
 		Prorate:    &True,
 	}
 	for _, subsc := range subs {
-		if subsc == nil || (mostRecentSub != nil && subsc.ID == mostRecentSub.ID) {
+		if subsc == nil || subsc.ID == "" || (mostRecentSub != nil && subsc.ID == mostRecentSub.ID) {
```

Comment:
I guess to find out you would log the customer id for which you can find empty subscription ids and then debug one customer in particular. List his subscriptions in the API and see how many are returned, compare it with the result you get using Stripe webapp.

Reply:
I agree. We need some logging in case a user has 

1. >1 active subscriptions
2. a subscription with an empty ID

From what I understand as soon as `subs` has a length > 1 something is wrong and we should know about it.

Reply:
I agree with both of you. And we do have logging. The issue this specific change is looking to resolve is that we somehow got subscriptions with empty IDs from Stripe. I can potentially add extra logging, so we can see those subs in more details but most of all I wanted to reduce the logspam for something we don't have control over.